src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionPool.cs

```diff
@@ -119,7 +119,11 @@ public HttpConnectionPool(HttpConnectionPoolManager poolManager, HttpConnectionK
             }
 
             _http2Enabled = _poolManager.Settings._maxHttpVersion >= HttpVersion.Version20;
-            _http3Enabled = _poolManager.Settings._maxHttpVersion >= HttpVersion.Version30 && (_poolManager.Settings._quicImplementationProvider ?? QuicImplementationProviders.Default).IsSupported;
+            // TODO: Replace with ILLink feature switch once https://github.com/dotnet/runtime/issues/44922 is finished
```

Comment:
Can we reword this comment? This won't be replaced by an ILLink feature switch. Instead, it will be replaced with an analyzer attribute that comes from that issue. ILLink is irrelevant here.

```suggestion
            // TODO: Replace with Platform-Guard Assertion Annotations once https://github.com/dotnet/runtime/issues/44922 is finished
```

Reply:
Done